common/app/routes/challenges/components/Classic-Modal.jsx

```diff
@@ -0,0 +1,74 @@
+import React, { PropTypes } from 'react';
+import { Button, Modal } from 'react-bootstrap';
+import PureComponent from 'react-pure-render/component';
+import FontAwesome from 'react-fontawesome';
+
+export default class ClassicModal extends PureComponent {
+
+  keyBindings(event) {
+    const map = Object.assign({}, this.props.keyMap);
```

Comment:
I see.

There are separate properties to detect if the ctrl/cmd key is used to modify the key press.

```
(e.ctrlKey || e.metaKey)
```

covers both mac and windows.